### PR TITLE
Scroll to top when tapping selected Tab

### PR DESCRIPTION
**Pull request by heytherewill**
## What's this?
This makes the scroll views auto-scroll when we tap on an already selected tab item

### Relationships
GO LOCO WEEK

## Why do we want this?
This behaviour is super common in apps. I find myself trying to do it all the time while testing.

## How is it done?
- Check if selected
- Check if View is able to scroll to top
- Scroll to top

### Why not another way?
This is straight forward enough
Only thing that needs figuring out is whether the calendar should scroll to top or to the current hour

### Side effects
Users saying _Nice_

## Review hints
Per commit

## :squid: Permissions
Only myself, but we need permission from @amulware and/or @maritoggl

### CODE REVIEW

**Comment by arturs105** [comment]
#4591

**Comment by amulware** [comment]
Is this how you propose to :pr::zero:? :D

**Comment by amulware** [comment]
You have my approval functionally, please make sure to reference the issue mentioned, and also check with @maritoggl.

File: Toggl.Daneel/ViewControllers/ReportsViewController.cs [hunk 1]
```diff
@@ -101,6 +102,12 @@ bool isWorkspaceNameTooLong(string workspaceName)
                 .DisposedBy(DisposeBag);
         }
 
+        public void ScrollToTop()
+        {
+            var indexPath = new CGPoint(0, -ReportsTableView.ContentInset.Top);
```
[the remark this answers is not in the document]
Comment (heytherewill) [reply Toggl.Daneel/ViewControllers/ReportsViewController.cs]: 🙈

**Comment by maritoggl** [comment]
I have no objections.

**Pull request merged**